src/api/static/intake/level1.yml

```diff
@@ -40,18 +35,13 @@ sources:
         type: str
         default: ""
       cache_type:
-        description: Type of caching to use with s3. Defaults to "simplecache"
-        type: str
-        default: "simplecache"
-      cache_dir:
-        description: Location to cache remote data locally.
+        description: Type of caching to use with s3. Defaults to "filecache"
```

Comment:
I actually think we should stick to using `simplecache` but without any `cache_dir` specified. Then we're not subtly filling up the user's disk

Reply:
Have swapped `filecache` back to `simplecache`, just checked the temp files from previous testing and they are still there so would pretty quickly fill up space if looking at many shots.